### PR TITLE
Add greenfield usage example

### PR DESCRIPTION
**Pull request by NicolasDorier**
Usage example documentation for Greenfield

### CODE REVIEW

**Comment by britttttk** [comment]
relates to #174

File: docs/GreenFieldExample.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# GreenField API example with cURL
+
+The [GreenField API](./API/Greenfield/v1/) (also available on your instance on `/docs`) allows you to operate BTCPay Server via an easy to use REST API.
+
+Note that you can partially generate clients in the language of your choice by using the [Swagger file](./API/Greenfield/v1/swagger.json).
```
Comment (pavlenex) [inline docs/GreenFieldExample.md:5]: Broken link, tests caught it.

Comment (dennisreimann) [reply docs/GreenFieldExample.md:5]: Those should be absolute links, becausee Vuepress doesn't know about these files.

**Comment by NicolasDorier** [comment]
fixed links

**Pull request merged**